library/postgres

```diff
@@ -4,52 +4,62 @@ Maintainers: Tianon Gravi <admwiggin@gmail.com> (@tianon),
              Joseph Ferguson <yosifkit@gmail.com> (@yosifkit)
 GitRepo: https://github.com/docker-library/postgres.git
 
+Tags: 10-beta1, 10
+Architectures: amd64, i386, ppc64le
```

Comment:
Just to verify: http://apt.postgresql.org/pub/repos/apt/dists/stretch-pgdg/10/
```

Name	Last Modified	Size	Type
Parent Directory/	 	-  	Directory
binary-amd64/	2017-May-18 14:15:00	-  	Directory
binary-i386/	2017-May-18 14:15:00	-  	Directory
binary-ppc64el/	2017-May-18 14:15:00	-  	Directory
source/	2017-May-18 14:15:00	-  	Directory
Contents-amd64.gz	2017-May-18 14:15:00	21.8K	application/x-gzip
Contents-i386.gz	2017-May-18 14:15:00	21.9K	application/x-gzip
Contents-ppc64el.gz	2017-May-18 14:15:00	21.9K	application/x-gzip
```
(same three arches :+1:)